browser/webpack.config.js

```diff
@@ -2,6 +2,9 @@ const path = require('path');
 
 module.exports = {
     entry: './conversion-tests/conversion_test.ts',
+    node: {
+      fs: 'empty'
+    },
```

Comment:
Why was this necessary?

Reply:
For some reason, we get an error that is close to "cannot resolve \"fs\" in @0x/dev-utils". After doing some reading, this is actually a bug in webpack and is fixed by adding this line. This is the issue I referred to find this solution: https://github.com/webpack-contrib/css-loader/issues/447. After reading the issue again, I'm going dig into this a bit more on a low-priority cycle, because it seems like there is a better way to fix this.